fxevent/console.go

```diff
@@ -45,10 +45,14 @@ func (l *ConsoleLogger) logf(msg string, args ...interface{}) {
 // LogEvent logs the given event to the provided Zap logger.
 func (l *ConsoleLogger) LogEvent(event Event) {
 	switch e := event.(type) {
-	case *LifecycleHookStart:
-		l.logf("START\t\t%s", e.CallerName)
-	case *LifecycleHookStop:
-		l.logf("STOP\t\t%s", e.CallerName)
+	case *LifecycleHookExecuting:
+		l.logf("HOOK %s\t\t%s executing (caller: %s)", e.Method, e.FunctionName, e.CallerName)
+	case *LifecycleHookExecuted:
+		if e.Err != nil {
+			l.logf("HOOK %s\t\t%s called by %s failed: %v", e.Method, e.FunctionName, e.CallerName, e.Err)
```

Comment:
should we include how long we took to fail here?

```suggestion
			l.logf("HOOK %s\t\t%s called by %s failed in %v: %v", e.Method, e.FunctionName, e.CallerName, e.Runtime, e.Err)
```